pkgs/top-level/python-packages.nix

```diff
@@ -4590,6 +4590,10 @@ self: super: with self; {
 
   freertos-gdb = callPackage ../development/python-modules/freertos-gdb { };
 
+  freesasa = callPackage ../development/python-modules/freesasa {
+    inherit (pkgs) freesasa;
+  };
```

Comment:
Any way to avoid this?
- #204303

Reply:
No, without this, a top-level package will be overwritten with a python library, and we should not pass pkgs as an argument.